py/nativeglue.c

```diff
@@ -348,6 +348,7 @@ const mp_fun_table_t mp_fun_table = {
     &mp_stream_readinto_obj,
     &mp_stream_unbuffered_readline_obj,
     &mp_stream_write_obj,
+    &mp_arg_parse_all_kw_array,
```

Comment:
Please move this up, say between `mp_get_stream_raise` and `mp_binary_get_size`.  And then increase `MP_FUN_TABLE_MP_TYPE_TYPE_OFFSET` by 1 (in `tools/mpy_ld.py`).